docker/prod/docker_staging.env

```diff
@@ -33,3 +33,13 @@ RDS_PASSWORD=x
 SECRET_KEY=some-secret-key
 SENTRY_URL=x
 SENDGRID_API_KEY=x
+
+CLUSTER=cluster
+CPU=0
+EXECUTION_ROLE_ARN=x
+MEMORY=0
+SUBNET_1=x
+SUBNET_2=x
+SUBNET_SECURITY_GROUP=x
```

Comment:
@RishabhJain2018 @KhalidRmb what about the values here? will this be changed at the time of deployment?

Reply:
yes, this will be modified at the time of deployment.